gpt_index/docstore.py

```diff
@@ -33,11 +33,20 @@ def serialize_to_dict(self) -> Dict[str, Any]:
             docs_dict[doc_id] = doc_dict
         return {"docs": docs_dict, "ref_doc_info": self.ref_doc_info}
 
-    def contains_index_struct(self, exclude_ids: Optional[List[str]] = None) -> bool:
+    def contains_index_struct(
```

Comment:
I still find it very confusing that a DocumentStore should even contain instances of IndexStruct.

Reply:
nit: Would be helpful to write a docstring for `DocumentStore` to describe what it actually contains.